opm/autodiff/NewtonIterationBlackoilInterleaved.cpp

```diff
@@ -153,8 +153,10 @@ namespace Opm
             const ParallelISTLInformation& info =
                 boost::any_cast<const ParallelISTLInformation&>( parallelInformation_);
             Comm istlComm(info.communicator());
+            // As we use a dune-istl with block size np the number of components
+            // per parallel is only one.
```

Comment:
I did not quite understand the comment, which probably means I do not understand what the argument to copyValuesTo() is supposed to be?